src/CounterFileRepository.php

```diff
@@ -51,24 +53,62 @@ public function addViewByUsername(Username $username): void
             FILE_APPEND
         );
 
-        $this->incrementViewsCount($username);
+        try {
+            $this->incrementViewsCount($username);
+        } catch (\Exception $exception) {
+            // Do not throw exception, because counter could be re-calculated
```

Comment:
We intentionally ignore issues with counter update, because we will be able to re-calculate counter if there will be problems.